File: Frontend/src/domain/functions/postIt.ts
import { randomWebSafeColor } from "./colors";

export function createPostIt(): PostIt {
  return {
    id: crypto.randomUUID(),
    x: 100,
    y: 100,
    w: 100,
    h: 60,
    color: randomWebSafeColor(),
    label: "New Item",
    version: 1
  };
}

File: Backend/src/Whiteboard.Domain/Models/PostIt.cs
namespace Whiteboard.Domain.Models;

public class PostIt{
    public Guid Id {get; set;}
    public int X {get; set; } 
    public int Y {get; set;} 
    public int W {get; set;}
    public int H {get; set;} 
    public string Color {get; set;} = "";
    public string Label {get; set;} = "";
    public long? Version {get; set;}
}